posts/draft.md

```diff
@@ -0,0 +1,174 @@
+# A deep dive into the MyST world
+
+> *This is a series of blog posts inviting you to join me in a little journey I have experienced in the last few weeks at the time to figure it out a nice story for **MyST**
+inside the **Nikola** world.*
+
+Hey, a lot of time without writing, you know... (lack of) time, but let's go back to the
+important things... or at least, interesting stuff to share with you!
+
+Recently, I have been involved with the [EBP](https://executablebooks.org/en/latest/)
+project and community (you can learn more about them [here](https://executablebooks.org/en/latest/about.html)). One of the major projects inside that community is the
+[MyST-Parser](https://github.com/executablebooks/MyST-Parser), a *rich* and *extensible*
+flavor of Markdown meant for technical documentation and publishing. It supports all thesyntax of CommonMark Markdown at its base but, in addition, it includes several
+extensions to CommonMark adding extra syntax features for technical writing, such as the
+roles and directives used by Sphinx in the rST world!
+
+When I looked into this project, I quickly realized it would interesting to explore how
+this flavor of Markdown could potentially integrate with the [Nikola](https://getnikola.com/)
+ecosystem. If you look into my previous blog posts, you will see I am a Nikola
+user and contributor and I am always interested in how we can improve the
+"blogging story" for those using Jupyter Notebooks. And the whole **EBP** project circle
+around an excel pipeline for publishing Jupyter Notebook being the Myst-Parser one of
+the central building blocks. They are even [interested](https://github.com/executablebooks/jupyter-book/issues/900)
+in a sane blog story for one of the key pieces the community has, the
+[Jupyter Book](https://jupyterbook.org/intro.html) project that indeed uses the
+Myst-Parser!
+
+Nikola, at the same time, has a long story providing support for
+multiple flavors of Markdown as Nikola plugins. Additionally, Nikola is probably unique
+in the Python static site generators arena because it is supporting Jupyter Notebook
+from the core package!
+
+So, all the pieces were asking me to explore this space and I could not resist ;-)
+
+# First impressions (someone has beaten me to it...)
+
+Ok, the first idea coming to my mind was to somehow implement MyST support in Nikola.
+But, hey, somebody probably already thought about this before me. So I made a search to
+see if this was not previously explored by others. And indeed that was the case!
+Roberto, the Nikola creator, wrote a plugin for MyST about a year ago (there is a brief
+blog post talking about it [here](https://ralsina.me/tr/es/weblog/posts/new-plugins-for-nikola.html)).
```

Comment:
Try to hyperlink more of the text so that it is easier for people to notice. For example, [the sentence here](https://executablebooks.org) is easier to notice, whereas this sentence [here](https://executablebooks.org) is harder to notice.
```suggestion
Roberto, the Nikola creator, wrote a plugin for MyST about a year ago (there is [a brief
blog post talking about it](https://ralsina.me/tr/es/weblog/posts/new-plugins-for-nikola.html)).
```